python/diagonal_b6/b6_test.py

```diff
@@ -141,32 +141,32 @@ def test_accessible_routes(self):
             self.assertLess(route.cost(), 500.0)
 
     def test_closest_from_point(self):
-        expression = b6.find_feature(b6.osm_node_id(STABLE_STREET_BRIDGE_SOUTH_END_ID)).closest("walk", 1000.0, b6.tagged("#amenity", "pub"))
+        expression = b6.find_feature(b6.osm_node_id(STABLE_STREET_BRIDGE_SOUTH_END_ID)).closest({"mode": "walk"}, 1000.0, b6.tagged("#amenity", "pub"))
         pub = self.connection(expression)
         self.assertEqual("The Lighterman", pub.get_string("name"))
 
     def test_closest_from_point_distance(self):
-        expression = b6.find_feature(b6.osm_node_id(STABLE_STREET_BRIDGE_SOUTH_END_ID)).closest_distance("walk", 1000.0, b6.tagged("#amenity", "pub"))
+        expression = b6.find_feature(b6.osm_node_id(STABLE_STREET_BRIDGE_SOUTH_END_ID)).closest_distance({"mode": "walk"}, 1000.0, b6.tagged("#amenity", "pub"))
         distance = self.connection(expression)
-        self.assertGreater(distance, 100.0)
-        self.assertLess(distance, 105.0)
+        self.assertGreater(distance, 128.0)
```

Comment:
this is b/c closest distance was using simple highway weights for walk / where as now it s walking weights